targets/curie_bsp/setup.py

```diff
@@ -0,0 +1,195 @@
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/parser\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/parser/js\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/parser/js/bc\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/parser/js/collections\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/parser/regexp\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-core/jrt\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-libm/include\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/jerry-libm\n'
+  out_put += 'subdir-cflags-y += -I$(PROJECT_PATH)/quark/jerryscript/targets/curie_bsp/include\n'
+  out_put += 'subdir-cflags-y += -Wno-error\n'
+  out_put += 'obj-$(CONFIG_QUARK_SE_ARC) += arc/\n'
+  out_put += 'obj-$(CONFIG_QUARK_SE_QUARK) += quark/\n'
+
+  file_to_be_created = open('Kbuild.mk', 'w+')
+  file_to_be_created.write(out_put)
+  file_to_be_created.close()
+  # Makefile
+  out_put =  'THIS_DIR   := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))\n'
+  out_put += 'T          := $(abspath $(THIS_DIR)/../..)\n\n'
+  out_put += 'PROJECT      := curie_jerry\n'
+  out_put += 'BOARD        := curie_101\n\n'
+  out_put += 'ifeq ($(filter curie_101, $(BOARD)),)\n'
+  out_put += '$(error The curie jerry sample application can only run on the curie_101 Board)\n'
+  out_put += 'endif\n'
+  out_put += 'BUILDVARIANT ?= debug\n'
+  out_put += 'quark_DEFCONFIG = $(PROJECT_PATH)/quark/defconfig\n'
+  out_put += 'arc_DEFCONFIG = $(PROJECT_PATH)/arc/defconfig\n'
+
+  out_put += '# Optional: set the default version\n'
+  out_put += 'VERSION_MAJOR  := 1\n'
+  out_put += 'VERSION_MINOR  := 0\n'
+  out_put += 'VERSION_PATCH  := 0\n'
+  out_put += 'include $(T)/build/project.mk\n'
+  file_to_be_created = open('Makefile', 'w+')
+  file_to_be_created.write(out_put)
+  file_to_be_created.close()
+
+  # Kbuild.mk in arc/
+  breadth_first_travel(curie_path + '/wearable_device_sw/projects/curie_bsp_jerry/arc/')
+  # Kbuild.mk in quark/
+  os.chdir(curie_path + '/wearable_device_sw/projects/curie_bsp_jerry/quark/')
+  out_put =  'obj-y += jerryscript/\n'
+  out_put += 'obj-y += main.o\n'
+  file_to_be_created = open('Kbuild.mk', 'w+')
+  file_to_be_created.write(out_put)
+  file_to_be_created.close()
+
+def main(curie_path, jerry_path):
+  pass
+  build_soft_links(curie_path, jerry_path)
+  create_kbuild_in_jerryscript(jerry_path)
+  create_makefile_in_curie(curie_path)
+
+
+if __name__ == '__main__':
+  if len(sys.argv) != 2:
+    print 'Usage:'
+    print sys.argv[0] + ' [full or relative path of Curie_BSP]'
+  else:
+    jerry_path = os.path.join(os.getcwd(), sys.argv[0])
+    npos = jerry_path.find('jerryscript')
+    jerry_path = jerry_path[0: npos] + 'jerryscript/'
```

Comment:
Does this builds the path for the `jerryscript` lib? If so what if someone names the repo dir `jerry`?
It would be better to calculate the path from this file something like this:
```python
file_dir = os.path.dirname(os.path.abspath(__file__)) # this is 'targets/curie_bsp/' 
jerry_path = os.path.join(file_dir, "..", "..")
```